types/d3-sankey/index.d.ts

```diff
@@ -1,10 +1,10 @@
-// Type definitions for D3JS d3-sankey module 0.11
+// Type definitions for D3JS d3-sankey module 0.12
```

Comment:
added this. lates is 0.12.3 but test cases won't let me add it

Reply:
DT uses the patch version as an incrementing counter of changes to a particular MAJOR.MINOR combination, so you can't match a specific MAJOR.MINOR.PATCH version.

The hope is that types do not change on patch versions since it would be an API break.